app/src/main/res/values/strings.xml

```diff
@@ -247,6 +245,7 @@
   <string name="registration_loggedout_dialog_title">You have been logged out</string>
   <string name="registration_loggedout_dialog_message">Please login again to continue</string>
   <string name="registration_loggedout_dialog_confirm">Okay</string>
+  <string name="registrationphone_country_code">+%1$s</string>
```

Comment:
Add the `translatable="false"` attribute?

Reply:
Good point, I'll make this change.